src/internal/kopia/upload.go

```diff
@@ -216,6 +216,7 @@ func (cp *corsoProgress) FinishedFile(relativePath string, err error) {
 		d.repoPath.String(),
 		d.repoPath.ShortRef(),
 		parent.ShortRef(),
+		d.repoPath.Item(),
```

Comment:
I don't think this will do quite what we want for all types. For OneDrive and SharePoint I think it will end up with the `.data` suffix on them because that's part of the item's name. It seems like at least for now we should try to shuffle this into service-specific code. Either the Stream should have another function to get this or details.go should be extracting it in a service-specific manner. The former is probably a bit more flexible (or at least closer to what's actually "generating" the ID?)

Reply:
fixed in https://github.com/alcionai/corso/pull/3174